src/api/routes/rosetta/construction.ts

```diff
@@ -267,15 +263,15 @@ export function createRosettaConstructionRouter(db: DataStore, chainId: ChainID)
     }
     const txSize: number = options.size;
 
-    let response: RosettaConstructionMetadataResponse;
+    let response = {} as RosettaConstructionMetadataResponse;
```

Comment:
A particular reason for this syntax ?

Reply:
yep, is a eslint config: `typescript-eslint/consistent-type-assertions use as instead`